test/cmdlineTests.sh

```diff
@@ -260,7 +260,7 @@ printTask "Running general commandline tests..."
         tdir=$(basename "${tdir}")
 
         inputFiles="$(find "${tdir}" -name 'input.*' -type f -exec printf "%s\n" "{}" \;)"
-        inputCount="$(echo "${inputFiles}" | wc -l)"
+        inputCount="$(find "${tdir}" -name 'input.*' -type f -exec printf "x" \; | wc -c)"
```

Comment:
`echo -n` does not really fix it (it always strips the newline even if the line is non-empty) so I had to resort to just repeating the `find` call.

Reply:
Do we really need all this `find` logic here? What about `$(ls ${tdir}/input.*)` and `echo "${inptuFiles}" | wc -w` - basically assume we do not use evil whitespace in paths?

Reply:
Well, `ls -1 ${tdir}/input.* 2> /dev/null | wc -w` would work. `ls -1 "${tdir}"/input.* 2> /dev/null | wc -l` would even work with whitespace in paths. I'll change it.